sources/packages/web/src/constants/message-constants.ts

```diff
@@ -9,3 +9,13 @@ export const PENDING_SIN_MESSAGE =
  */
 export const INVALID_SIN_MESSAGE =
   "StudentAid BC has found an issue with your account information. Please ensure your account information matches CRA and log back in. If you have further issues please contact us.";
+
+/**
+ * Ministry Name
+ */
+export const MINISTRY_SHORTNAME = "PSFS";
+
+/**
+ * Ministry Fullname
+ */
+export const MINISTRY_NAME = "Ministry of Post Secondary and Future Skills";
```

Comment:
I am not sure if it would be part of the ticket, but this const can also be used inside the form definitions also.
If we have confirmations that the same name would be used for the legal disclaimers and also the name displayed on the home page (and other pages), we can have it all centralized under the same place.

Reply:
Should I do this as well?

Reply:
You can finish this PR and if we have the confirmation that "the same name would be used for the legal disclaimers", I would recommend doing it right after this PR (it can be the same ticket).
The main point is just, if the Ministry name changes again, a developer looking in the references of the const will be able to easily figure out all points impacted.